bean/internal/driver/template/login.html

```diff
@@ -1,10 +1,23 @@
-{{ define "body" }}
-  <form name="login" method="post">
-    <input type="text" name="username" placeholder="Username" />
-    <input type="password" name="password" placeholder="Password" />
-    <input type="submit" value="Login" />
-  </form>
-  {{ if .Error }}
-    <p>{{ .Error }}</p>
-  {{ end }}
+{{ define "navbar" }}
+<a href="/">Home</a>
+&nbsp;·&nbsp;
+<a>Get started</a>
+{{ end }}
```

Comment:
I'm not a fan of repeating this code

But it does seem like the simpler option right now